### PR TITLE
Versal - platform updates

### PR DESCRIPTION
**Pull request by ldts**
reference: https://github.com/OP-TEE/optee_os/pull/5426

### CODE REVIEW

File: core/arch/arm/plat-versal/link.mk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+include core/arch/arm/kernel/link.mk
+
+all: $(link-out-dir)/tee-raw.bin
```
Comment (jforissier) [inline core/arch/arm/plat-versal/link.mk:3]: Shouldn't that be `all: $(link-out-dir)/uTee`? In other words, why should `tee-raw.bin` be generated by default but not `uTee`?

**Review by jforissier — COMMENTED** [review]
> `Increase the number of regions`

Why, or perhaps more precisely, why by default?

File: core/arch/arm/plat-versal/link.mk (hunk 1 above)
Comment (ldts) [reply core/arch/arm/plat-versal/link.mk:3]: sorry yes. we dont really need uTee

I think I can just 

`all: $(link-out-dir)/tee-raw.bin  `

Comment (jforissier) [reply core/arch/arm/plat-versal/link.mk:3]: OK, so with just the above two lines (include + all:) this commit is:

`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by ldts** [comment]
> > `Increase the number of regions`
> 
> Why, or perhaps more precisely, why by default?

sorry didnt notice your question: yes so just as done on other platforms - probably more than what is actually needed but I had to increase it while developing some of the drivers (IIRC when I added support for the GPIO driver)...so picked 24 as this value seemed the norm and allowed my drivers to be mapped.

**Comment by jforissier** [comment]
> > > `Increase the number of regions`
> > 
> > 
> > Why, or perhaps more precisely, why by default?
> 
> sorry didnt notice your question: yes so just as done on other platforms - probably more than what is actually needed but I had to increase it while developing some of the drivers (IIRC when I added support for the GPIO driver)...so picked 24 as this value seemed the norm and allowed my drivers to be mapped.

Alright, makes sense of course. Perhaps you could add a few words in the commit description, for example mentioning that the new value is a reasonable number to use when enabling a few more drivers than are currently available upstream?

For that commit:
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by ldts** [comment]
For reference, there is now an easy way to build a release besides Foundries.io v88 https://foundries.io/products/releases/88/
https://github.com/OP-TEE/build/pull/590

**Comment by ldts** [comment]
@jforissier updated with tags. Added additional information on both commits.

**Comment by jforissier** [comment]
> @jforissier updated with tags. Added additional information on both commits.

You forgot the Ack on the last one :wink:

**Comment by ldts** [comment]
argh sorry. doing it now

**Pull request merged**